src/main/webapp/script.js

```diff
@@ -35,9 +36,18 @@ function initMap() {
 
     //change the map to the input location
     const geocoder = new google.maps.Geocoder();
-    document.getElementById("submit").addEventListener("click", () => {
+    document.getElementById("submitLocation").addEventListener("click", () => {
         geocodeAddress(geocoder, map);
     });
+
+    //resize the radius using the var "rad" which is used to draw the circle and create the nearbySearch results
+    //the issue is, when rad = an int, it works. When rad = doc.get, it does not
+    document.getElementById("submitRadius").addEventListener("click", () => {
+          //rad = document.getElementById("resize").value;
```

Comment:
`rad = document.getElementById("resize").value;`
When you use this line of code, it actually does resize correctly.
What's actually going wrong is that the circle doesn't redraw. If you continue testing with different values in "resize" input and generate results, you can see that the radius is indeed changing. I think what happens is that once the radius is resized, `createCityCircle()` isn't being called properly (or sth is wrong there).
It seems like `nearbySearch()` works with rad though.

Reply:
In the JS console, clicking on "resize" generates this stack trace:
`InvalidValueError: setRadius: not a number`
_.qe | @ | js?key=AIzaSyBnHjJrX…=places&v=weekly:78
-- | -- | --
   _.te | @ | js?key=AIzaSyBnHjJrX…=places&v=weekly:79
   (anonymous) | @ | js?key=AIzaSyBnHjJrX…places&v=weekly:101
   _.M.setValues | @ | js?key=AIzaSyBnHjJrX…places&v=weekly:222
   _.bj | @ | js?key=AIzaSyBnHjJrX…places&v=weekly:141
   createCityCircle | @ | script.js:183
   (anonymous) | @ | script.js:49

Reply:
It's possible that the rad variable needs to be sanitized? Since the input may somehow be inferred as a String type and cause an error when `createCityCircle()` tries to set the radius value.

Reply:
@yinlow63 not sure if you'd be familiar with what might be going on here